tensorflow/python/ops/image_ops_impl.py

```diff
@@ -2937,7 +2968,7 @@ def reducer(x):
 
 
 @tf_export('image.ssim')
-def ssim(img1, img2, max_val):
+def ssim(img1, img2, max_val, **kwargs):
```

Comment:
Why *kwargs instead of specifying and documenting the new parameters?

Reply:
Hello. I though that kwargs might be better since most people do not change them. Also, this is the case in skimage.measure.compare_ssim. However, I think that having them as separate variables would also be ok.

Reply:
@alextp I think that explicitly naming the variables would also be great. I originally suggested kwargs because that is how K1 and K2 are stored in the skimage implementation.